pyproject.toml

```diff
@@ -6,18 +6,19 @@ profile = "black"
 
 [tool.poetry]
 name = "toogoodtogo-ha-mqtt-bridge"
-version = "1.10.1"
+version = "2.0.0"
 description = ""
 authors = ["Max Winterstein <github@winterstein.mx>"]
 
 [tool.poetry.dependencies]
 python = "^3.7"
 paho-mqtt = "^1.5.1"
 dynaconf = "^3.1.2"
-tgtg = "0.10.0"
+tgtg = "0.10.1"
 coloredlogs = "^15.0"
 tenacity = "^6.3.1"
 arrow = "^0.17.0"
+croniter = "^1.1.0"
```

Comment:
Any reason why you don't push the `poetry.lock` file? 🤔

Reply:
I don't know how to work with poetry... so, this line in pyproject.toml was manually added.

Reply:
okay, no problem. Still thinking about replacing the whole poetry stuff. Will add some fix if needed afterwards 👍